lib/pure/collections/tables.nim

```diff
@@ -1619,6 +1619,8 @@ proc `==`*[A, B](s, t: OrderedTable[A, B]): bool =
 
   if s.counter != t.counter:
     return false
+  if s.counter == 0 and t.counter == 0:
```

Comment:
```suggestion
  if s.counter == 0:
```

Reply:
Yes, I'm aware of that :)

But I wanted to be more explicit (than necessary) just to convey what is the purpose of that part of the code (if you don't read the above lines).